<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;华文仿宋&quot;;</string>
     </property>
     <property name="text">
      <string>接收数据：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="reciveLE">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>暂无数据</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="ipLE">
       <property name="placeholderText">
        <string>请输入IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;华文仿宋&quot;;</string>
       </property>
       <property name="text">
        <string>端口号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLE">
       <property name="placeholderText">
        <string>默认端口：0471</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;华文仿宋&quot;;</string>
     </property>
     <property name="text">
      <string>发送数据：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="sendLE">
     <property name="placeholderText">
      <string>在此输入数据</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="startServerBtn">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>开启服务器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeServerBtn">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>关闭服务器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>清除日志</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
